Test Monitor/OEE/OEE.fpg

```diff
@@ -114,733 +84,80 @@
+"bindings": [{"value": [{"type":"notebook","binding":"{\"name\":\"_shared/reports/OEE Health.ipynb\",\"executionInterval\":86400,\"version\":2,\"output\":\"availability_scalar\",\"params\":[{\"name\":\"group_by\",\"value\":\"Day\"},{\"name\":\"results_filter\",\"value\":\"startedWithin <= \\\"30.0:0:0\\\"\"}]}"}]}]},
+{"name": "tile8", "type": "progress", "x": "581", "y": "0", "width": "575", "height": "325.6875", "attributes": [{"prefix": "tile"}, {"title": {"localizationKey":"tile8.title"}}, {"datasource": "%7B%22type%22%3A%22notebook%22%2C%22binding%22%3A%22%7B%5C%22name%5C%22%3A%5C%22_shared/reports/OEE%20Health.ipynb%5C%22%2C%5C%22executionInterval%5C%22%3A86400%2C%5C%22version%5C%22%3A2%2C%5C%22output%5C%22%3A%5C%22performance_scalar%5C%22%2C%5C%22params%5C%22%3A%5B%7B%5C%22name%5C%22%3A%5C%22group_by%5C%22%2C%5C%22value%5C%22%3A%5C%22Day%5C%22%7D%2C%7B%5C%22name%5C%22%3A%5C%22results_filter%5C%22%2C%5C%22value%5C%22%3A%5C%22startedWithin%20%3C%3D%20%5C%5C%5C%2230.0%3A0%3A0%5C%5C%5C%22%5C%22%7D%5D%7D%22%7D"}, {"progressDisplayFormat": "horizontal"}, {"denominator": "100"}, {"navigationURLEnabled": "false"}, {"useFilter": "false"}, {"zIndex": "13"}, {"columnRow": "%5B3%2C0%5D"}, {"gridSpan": "%5B3%2C2%5D"}],
+"bindings": [{"value": [{"type":"notebook","binding":"{\"name\":\"_shared/reports/OEE Health.ipynb\",\"executionInterval\":86400,\"version\":2,\"output\":\"performance_scalar\",\"params\":[{\"name\":\"group_by\",\"value\":\"Day\"},{\"name\":\"results_filter\",\"value\":\"startedWithin <= \\\"30.0:0:0\\\"\"}]}"}]}]},
+{"name": "tile12", "type": "progress", "x": "1160", "y": "0", "width": "575", "height": "325.6875", "attributes": [{"prefix": "tile"}, {"title": {"localizationKey":"tile12.title"}}, {"datasource": "%7B%22type%22%3A%22notebook%22%2C%22binding%22%3A%22%7B%5C%22name%5C%22%3A%5C%22admin/reports/new/OEE%20Health.ipynb%5C%22%2C%5C%22executionInterval%5C%22%3A86400%2C%5C%22version%5C%22%3A2%2C%5C%22output%5C%22%3A%5C%22quality_scalar%5C%22%2C%5C%22params%5C%22%3A%5B%7B%5C%22name%5C%22%3A%5C%22group_by%5C%22%2C%5C%22value%5C%22%3A%5C%22Day%5C%22%7D%2C%7B%5C%22name%5C%22%3A%5C%22products_filter%5C%22%2C%5C%22value%5C%22%3A%5C%22%5C%22%7D%2C%7B%5C%22name%5C%22%3A%5C%22results_filter%5C%22%2C%5C%22value%5C%22%3A%5C%22startedWithin%20%3C%3D%20%5C%5C%5C%2230.0%3A0%3A0%5C%5C%5C%22%5C%22%7D%5D%7D%22%7D"}, {"progressDisplayFormat": "horizontal"}, {"denominator": "100"}, {"navigationURLEnabled": "false"}, {"useFilter": "false"}, {"zIndex": "14"}, {"columnRow": "%5B6%2C0%5D"}, {"gridSpan": "%5B3%2C2%5D"}],
+"bindings": [{"value": [{"type":"notebook","binding":"{\"name\":\"admin/reports/new/OEE Health.ipynb\",\"executionInterval\":86400,\"version\":2,\"output\":\"quality_scalar\",\"params\":[{\"name\":\"group_by\",\"value\":\"Day\"},{\"name\":\"products_filter\",\"value\":\"\"},{\"name\":\"results_filter\",\"value\":\"startedWithin <= \\\"30.0:0:0\\\"\"}]}"}]}]},
```

Comment:
The notebook specified on this tile is different than all the other tiles:
admin/reports/new/OEE Health.ipynb vs _shared/reports/OEE Health.ipynb

I'm guessing that is not intentional?